src/inmanta/module.py

```diff
@@ -458,6 +476,52 @@ def _should_install_module() -> bool:
             return self.install(project, module_spec)
         return installed
 
+    def _get_constraints(self, module_name: str, module_spec: List[InmantaModuleRequirement]) -> str:
+        """
+        Returns the constraints on a given inmanta module as a string.
+
+        :param module_name: The name of the module.
+        :param module_spec: List of inmanta requirements in which to look for the module.
+        """
+        constraints_on_module: List[str] = [str(req) for req in module_spec if module_name == req.key and req.specs]
+        from_constraints = f" (from constraints {' '.join(constraints_on_module)})" if constraints_on_module else ""
```

Comment:
Return the empty string if `not constraints_on_module`?

Reply:
Yes, if there are no constraints it will read:
```bash
"Installing module std (v1)
```
I think it's better to be explicit and to go with:
```bash
"Installing module std (v1) (with no version constraint)
```

Reply:
Oh wow, not sure how I missed that. My comment was actually a suggestion to do that. I guess I read the line to quickly, my apologies.

Reply:
I have no strong preference as to the explicit "no version constraint".